app/controllers/schools/estimate_participants_controller.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class Schools::EstimateParticipantsController < Schools::BaseController
+  # who has permissions to view this ?
```

Comment:
School induction tutors. That's being checked in the base controller which this controller extends